tasks/section5.yml

```diff
@@ -111,15 +129,19 @@
       - rule_5.1.6
 
 - name: "SCORED | 5.1.6 | PATCH | Ensure permissions on /etc/cron.monthly are configured"
-  command: /bin/true
+  file:
+      dest: /etc/cron.monthly
+      owner: root
+      group: root
+      mode: 0600
   tags:
       - level1
       - level2
       - patch
       - rule_5.1.6
 
 - name: "SCORED | 5.1.7 | AUDIT | Ensure permissions on /etc/cron.d are configured"
-  command: /bin/true
+  shell: stat -L -c "%a %u %g" /etc/cron.d | egrep ".00 0 0"
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*